src/components/common/ColonyMembers/TotalReputation/TotalReputation.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import Decimal from 'decimal.js';
+
+import Heading from '~shared/Heading';
+import Numeral from '~shared/Numeral';
+
+import { getFormattedTokenValue } from '~utils/tokens';
+import {
+  ROOT_DOMAIN_ID,
+  DEFAULT_TOKEN_DECIMALS,
+  ADDRESS_ZERO,
+} from '~constants';
+import { useColonyContext } from '~hooks';
+import { useGetUserReputationQuery } from '~gql';
+
+import styles from './TotalReputation.css';
+
+const displayName = 'common.ColonyMembers.TotalReputation';
+
+const MSG = defineMessages({
+  totalReputationTitle: {
+    id: `${displayName}.totalReputationTitle`,
+    defaultMessage: 'Total reputation in the team',
+  },
+});
+
+type Props = {
+  selectedDomainId: number;
+};
+
+const TotalReputation = ({ selectedDomainId }: Props) => {
+  const { colony } = useColonyContext();
+  const { data: totalReputation } = useGetUserReputationQuery({
+    variables: {
+      input: {
+        walletAddress: ADDRESS_ZERO,
+        colonyAddress: colony?.colonyAddress || '',
+        domainId: selectedDomainId || ROOT_DOMAIN_ID,
+      },
+    },
+    fetchPolicy: 'cache-and-network',
+  });
+
+  const formattedTotalDomainRep = getFormattedTokenValue(
+    new Decimal(totalReputation?.getUserReputation || '0').abs().toString(),
+    colony?.nativeToken?.decimals || DEFAULT_TOKEN_DECIMALS,
+  );
```

Comment:
Could you try passing `colony?.nativeToken?.decimals || DEFAULT_TOKEN_DECIMALS` as a `decimals` prop of `Numeral` component? If it works, it should save you the manual formatting here

Reply:
Sure, that works well, let me know if thats what you had in mind.